config/environments/deployed_development.rb

```diff
@@ -0,0 +1,107 @@
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
If we are adding many of them, can we remove comments from them to make them easier to compare? 

Actually, before we do. Is this any different to `production` one? And is it any different to staging?

Reply:
The only difference with production is that has some ENV["IGNORE_SECRETS_FOR_BUILD"]

No difference with staging at the moment, but I expect that to change